attractions/button/button.svelte

```diff
@@ -1,3 +1,7 @@
+<!-- TODO: Decide whether to write out all tag names or leave them as null
```

Comment:
Yeah, I think it's best to leave them to be specified by users. That way:
1. We don't have to document new things
2. The end user code gets a little more readable since it's evident where the custom elements are coming from

Reply:
And then maybe we could distribute a simple JS file that registers all the components with an opinionated name, such as `a-button` as in this example.

Reply:
Actually both way are somewhat incompatible. To remove the names, we would have to specify `<svelte:options tag={null} />`, and then to register the name we use `customElements.register('a-button', attractions.Button)`. I'm not against the idea of distributing 2 versions, but it would be additional writing we need to do, and the 2nd version is only useful for people using another framework (Vue. for instance).
I would imagine that including one script file from a CDN and using ready components right away would be the most common use-case (for the plain JS users) as it is the fastest for prototyping and doesn't need much additional work. The other approach will require polluting the global window (with the variable `attractions`) and users would need to write a lot of duplicated code to register all components they need

Reply:
I genuinely don't think there will be a lot of components. To me it seems like you'll use custom elements when you just want one or two components from this library, if you want to actually build an interface, you employ Svelte itself. So this code to register the elements won't be much

Reply:
Yes, but if you are using the library from a CDN without Svelte or some other framework, you probably need a very quick prototype and don't really want to spend time registering all components you need, wondering why some aren't working just to find out you forgot to register some of them. It also requires remembering the API and knowing where to find the components to be registered (the global `attractions` object), in addition to polluting the global namespace. You probably wouldn't even want to write a lot of JavaScript.
The main point is that it does no harm to register them, and anyone using a different framework or simply doesn't like the names can rename them

Reply:
okay, fine, let's register them with the `a-` prefix